src/views/Policies/PolicyView.spec.ts

```diff
@@ -0,0 +1,51 @@
+import VueRouter from 'vue-router'
+import Vuex from 'vuex'
+import { createLocalVue, mount } from '@vue/test-utils'
+
+import Store from '@/store/index'
+
+import PolicyView from './PolicyView.vue'
+
+const localVue = createLocalVue()
+
+localVue.use(VueRouter)
+localVue.use(Vuex)
+
+const router = new VueRouter()
+
+async function createWrapper(props = {}) {
+  const store = new Vuex.Store(Store())
+
+  await store.dispatch('fetchPolicies')
+
+  return mount(PolicyView, {
+    localVue,
+    router,
+    store,
+    propsData: props,
+  })
+}
+
+describe('PolicyView', () => {
+  test('renders default view correctly', async () => {
+    const wrapper = await createWrapper({ policyPath: 'circuit-breakers' })
+
+    // For some magical reason, this exact arrangement is necessary to wait for the component to completely load. Yikes.
```

Comment:
Can't we just await for some elements to be visible (like puppeteer `waitForSelector`), or [flush promises](https://v1.test-utils.vuejs.org/guides/testing-async-components.html#asynchronous-behavior-outside-of-vue)?

Reply:
With Vue 3/Vue test utils 2, we can indeed simply use `flushPromises` which test utils handily provides since then. The current approach is admittedly not great, but then the upgrade is very close on the horizon. I should also just annotate that intend to use `flushPromises` which I did in a similar case elsewhere.